### DISCUSSION TITLE
setOn nutzen um mehrere DPs auf true zu schalten

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2018] [Q&A]
Ist mit setOn (Datenpunkt-Eigenschaften) ein Einschalten mehrerer DPs möglich?
{"setOn":{"{0_userdata.0.DP.dp1}":true,"{0_userdata.0.DP.dp2}":true}}

Hintergrund:
https://forum.iobroker.net/topic/59567/jarvis-trigger-element-auf-mehrere-datenpunkte-anwenden

**Comment by Zefau** [comment]
Nein das geht nicht

**Reply by mcuiobroker** [reply]
Wäre es denn dort sinnvoll?
-> Feature Request.

**Reply by Zefau** [reply]
Gibt schon einen Feature Request: https://github.com/Zefau/ioBroker.jarvis/issues/695
